espflash/src/targets/esp32s2.rs

```diff
@@ -72,11 +72,23 @@ impl Esp32s2 {
 
 impl ReadEFuse for Esp32s2 {
     fn efuse_reg(&self) -> u32 {
-        0x3f41_a000
+        0x3F41_A000
```

Comment:
What's up with the address? The TRM has 0x3ffc1a000
![image](https://github.com/user-attachments/assets/9861ef1d-6a4d-4b3a-9abc-f647ce97165d)

Reply:
These addresses are taken from `esptool`, no idea why there's no consistency with this stuff.

Reply:
Apologies, I was looking at TRM v1.1, which has since been corrected.